.rubocop.yml

```
@@ -45,3 +45,11 @@ Style/Documentation:
 Style/GuardClause:
   Exclude:
     - 'app/controllers/api/v1/testimonials_controller.rb'
+
+MultipleExpectations:
+  Exclude:
+    - spec/**/*
+
+MultipleMemoizedHelpers:
+  Exclude:
+    - spec/**/*
```

You can configure your IDE to add automatically en empty line at the end of file:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files
```suggestion
    - spec/**/*
    
```